test/functional/page_objects/settings_page.ts

```diff
@@ -375,14 +375,33 @@ export function SettingsPageProvider({ getService, getPageObjects }: FtrProvider
       return indexPatternId;
     }
 
-    async setIndexPatternField({ indexPatternName = 'logstash-', expectWildcard = true } = {}) {
+    async setIndexPatternField(indexPatternName = 'logstash-*') {
       log.debug(`setIndexPatternField(${indexPatternName})`);
       const field = await this.getIndexPatternField();
       await field.clearValue();
-      await field.type(indexPatternName, { charByChar: true });
+      if (
+        indexPatternName.charAt(0) === '*' &&
+        indexPatternName.charAt(indexPatternName.length - 1) === '*'
+      ) {
+        // this is a special case when the index pattern name starts with '*'
+        // like '*:makelogs-*' where the UI will not append *
+        await field.type(indexPatternName, { charByChar: true });
+      } else if (indexPatternName.charAt(indexPatternName.length - 1) === '*') {
+        // the common case where the UI will append '*' automatically so we won't type it
+        const tempName = indexPatternName.slice(0, -1);
+        await field.type(tempName, { charByChar: true });
+      } else {
+        // case where we don't want the * appended so we'll remove it if it was added
+        await field.type(indexPatternName, { charByChar: true });
+        const tempName = await field.getAttribute('value');
+        if (tempName.length > indexPatternName.length) {
+          await field.type(browser.keys.RIGHT, { charByChar: true });
+          await field.type(browser.keys.BACK_SPACE, { charByChar: true });
+        }
```

Comment:
the 2 commands with `browser.keys.RIGHT` and `browser.keys.BACK_SPACE` might be able to be replaced with `browser.keys.DELETE` ?  I'm not sure that's a valid key, just thought of it.

Reply:
You can do `await field.type([browser.keys.RIGHT, browser.keys.BACK_SPACE], { charByChar: true });` as well

Reply:
@LeeDr you can simply do `await field.type([browser.keys.DELETE]);`, tested in both FF and Chrome